doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>in this notebook we -&gt; in this notebook, we</p><p> </p><p>divide in 20 -&gt; divide into 20</p><p> </p><p>found here -&gt; link not working</p><p> </p><p>train dataset -&gt; training dataset</p><p> </p><p>consider adding python modules between backticks</p><p> </p><p>link abstract of the papers and not the actual pdfs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>